justfile

```diff
@@ -7,4 +7,10 @@ install-contracts:
   forge install
 
 install-eip712sign:
-  go install github.com/base-org/eip712sign@v0.0.3
+  #!/usr/bin/env bash
+  REPO_ROOT=`git rev-parse --show-toplevel`
+  cd $REPO_ROOT
+  mkdir -p bin || true
+  cd lib/eip712sign
```

Comment:
Only comment is the order of `install: install-contracts install-eip712sign` now matters since this needs the submodules installed, worth documenting in a comment